### PR TITLE
First pass at CSM 1.3.2 patch instructions, and add upgrade steps for CASMHMS-5911

### PR DESCRIPTION
**Pull request by rsjostrand-hpe**
# Description

First pass at CSM 1.3.2 patch instructions. I needed to insert a special step for to modify the customizations.yaml before the the updated manifests where deployed. 
    
CASMHMS-5911: Added steps to make the needed changes for the cray-hms-hmcollector chart to be accessible at `hmcollector.hmnlb.$SYSTEM_NAME.$SITE_DOMAIN` on upgrades to CSM 1.3.2. Either from directly from CSM 1.2 or CSM 1.3.1.

Test of the yq statement:
```
ncn-m001:~/rsjostrand/collector #    kubectl get secrets -n loftsman site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d > "customizations.yaml"
ncn-m001:~/rsjostrand/collector # cp customizations.yaml customizations.yaml.bak
ncn-m001:~/rsjostrand/collector #    yq w -i "customizations.yaml" 'spec.kubernetes.services.cray-hms-hmcollector.hmcollector_external_hostname' 'hmcollector.hmnlb.{{ network.dns.external }}'
```
Verify it updated customizations.yaml as expected:

```
ncn-m001:~/rsjostrand/collector # diff customizations.yaml.bak customizations.yaml
462a463
>         hmcollector_external_hostname: hmcollector.hmnlb.{{ network.dns.external }}
ncn-m001:~/rsjostrand/collector # grep hmcollector_external_hostname customizations.yaml -C 2
      cray-hms-hmcollector:
        hmcollector_external_ip: '{{ network.netstaticips.hmn_api_gw }}'
        hmcollector_external_hostname: hmcollector.hmnlb.{{ network.dns.external }}
      cray-hms-meds:
        ntpserver_host: time-hmn:123
```

Additional testing was performed in the accompanying manifest PR to the CSM repo: https://github.com/Cray-HPE/csm/pull/1828

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by shunr-hpe — APPROVED** [review]

File: upgrade/1.3.2/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+# CSM 1.3.2 Patch Installation Instructions
+
+## Introduction
+
+This document guides an administrator through the patch update to Cray Systems Management `v1.3.2` from `v1.3.1`.
```
Comment (nrockershousen) [inline upgrade/1.3.2/README.md:5]:
```suggestion
This document guides an administrator through the patch update to Cray Systems Management (CSM) `v1.3.2` from `v1.3.1`.
```

File: upgrade/1.3.2/README.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+## Update `customizations.yaml`
+
+1. Retrieve `customizations.yaml` from the `site-init` secret:
+
+   ```bash
+   kubectl get secrets -n loftsman site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d > "customizations.yaml"
+   ```
+
+1. Add customizations for the `cray-hms-hmcollector` helm chart:
```
Comment (nrockershousen) [inline upgrade/1.3.2/README.md:63]:
```suggestion
1. Add customizations for the `cray-hms-hmcollector` Helm chart:
```

**Review by nrockershousen — APPROVED** [review]

**Review by rustydb — CHANGES_REQUESTED** [review]
Requires this to merge: https://github.com/Cray-HPE/csm/pull/1828#pullrequestreview-1294051336

Marking as "needs work" to block someone from merging this in the meantime.

File: upgrade/1.3.2/README.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
+# CSM 1.3.2 Patch Installation Instructions
+
+## Introduction
+
+This document guides an administrator through the patch update to Cray Systems Management (CSM) `v1.3.2` from `v1.3.1`.
+If upgrading from CSM `v1.2.2` directly to `v1.3.2`, follow the procedures described in [Upgrade CSM](../README.md) instead.
+
+## Bug Fixes and Improvements
+
+* Fixed a scale issue with the `/v1/hardware` API in the System Layout Service (SLS) causing the service become deadlocked.
```
Comment (johren-hpe) [inline upgrade/1.3.2/README.md:10]:
```suggestion
* Fixed a scale issue with the `/v1/hardware` API in the System Layout Service (SLS) causing the service to become deadlocked.
```

File: upgrade/1.3.2/README.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+1. Download and extract the CSM `v1.3.2` release to `ncn-m001`.
+
+   See [Download and Extract CSM Product Release](../../update_product_stream/index.md#download-and-extract).
+
+1. Set `CSM_DISTDIR` to the directory of the extracted files.
+
+   **IMPORTANT**: If necessary, change this command to match the actual location of the extracted files.
+
+   ```bash
+   CSM_DISTDIR="$(pwd)/csm-1.3.1"
```
Comment (johren-hpe) [inline upgrade/1.3.2/README.md:40]:
```suggestion
   CSM_DISTDIR="$(pwd)/csm-1.3.2"
```

**Review by johren-hpe — CHANGES_REQUESTED** [review]

**Review by gbaker-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**